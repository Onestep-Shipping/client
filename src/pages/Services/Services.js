import './Services.css';

import Header from '../../components/Header/Header.js';
import React from 'react';

const Services = () => {
  return (
    <div className="homepage-container">
      <Header />
      <div className="body-container2">
        Services
      </div>
    </div>
  );
};

export default Services;
