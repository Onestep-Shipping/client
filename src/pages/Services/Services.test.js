import React from 'react';
import Services from './Services';
import { shallow } from 'enzyme';

let wrapper = shallow(<Services />);

describe('Services', () => {
  it('should render the Services Component correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});