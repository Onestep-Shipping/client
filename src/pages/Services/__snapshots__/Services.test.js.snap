// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services should render the Services Component correctly 1`] = `
<div
  className="homepage-container"
>
  <Header />
  <div
    className="body-container2"
  >
    Services
  </div>
</div>
`;
